.github/config.yml

```diff
@@ -0,0 +1,53 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/code_of_conduct.md).🙌🏻
+
+  You may submit a PR on the same if you like!
+  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) while sending PRs.
+  If you want to report a bug🐞, please follow our [bug_report.md](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) template.
+  Also please make sure you include steps to reproduce it and be patient we will get back to you as soon as we can.👀
+
```

Comment:
Should we also add the link to the calendar? For contributors to join project specific open sesssions eg. Mentorship Open Session?